extensions/interactions/ImageClickInput/ImageClickInput.py

```diff
@@ -59,6 +59,7 @@ class ImageClickInput(base.BaseInteraction):
         'options': {
             'x_axis_label': 'Answer',
             'y_axis_label': 'Count',
+            'show_addressed_info': False,
```

Comment:
Btw, changing the name isn't what I meant. I was just wondering why the interactions using BarChart even needed this option to be specified at all. That is, what would the visualization do if the value was True instead?

Reply:
Hmm, right now True would do nothing so it's definitely useless. The main reason I added (and renamed the variable) is because that's the simplest way to get the PR passing. As for the reasoning to keep it, I figured we may be able to give additional on-hover information to the bars (like an "address now" link), since how things are "addressed" still makes sense here. Do you see that as an option?

Reply:
I've put some comments in the files to argue why they're there, btw.

Reply:
Ah, I see. Yes, that's fine, as long as it's a decision that gets reinforced for any new visualization/interaction. So I would suggest making sure that the validation tests in the backend check that every 'options' field has a 'show_addressed_info' subfield, and also a test to ensure that the specifications for each visualization option (the ones that say "this thing must have the options x_axis_label and y_axis_label") don't include a string that matches "show_addressed_info".